app-frontend/src/app/components/scenes/sceneDetailModal/sceneDetailModal.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../../assets/styles/sass/settings/_colors.scss";
+
+input.form-control.scene-detail-field {
+    height: 32px;
+    border: none;
+    border-bottom: 3px dotted $gray-lightest;
+    color: $brand-tertiary;
+    font-style: italic;
+}
+
+a.btn.dropdown-label.acq-date {
+    color: $brand-tertiary;
+    font-style: italic;
+    border: none;
+    border-bottom: 3px dotted $gray-lightest;
+}
+
+a.btn.dropdown-label.acq-date:hover {
```

Comment:
I would move the bottom borders from the buttons within the btn-group to the btn-group itself.

Also, it would be good to make these selectors less specific. There are a few options here: `.meta-list .acc-date-label`, `.meta-list .acc-date-toggle`, etc. may be the simplest.
Very specific selectors are often brittle in that they can be broken easily by insignificant changes. Here, for example, if someone changed the dropdown elements to buttons, the styles would disappear, and below the opposite is true.